cli/bptest/cmd.go

```diff
@@ -145,3 +146,14 @@ var initCmd = &cobra.Command{
 		return initTest(initTestName)
 	},
 }
+
+var lintCmd = &cobra.Command{
+	Use:   "lint",
+	Short: "Lint metadata.yaml file.",
+	Long:  "Lint metadata.yaml file according to requirements in go/blueprints-adc.",
+	Args:  cobra.NoArgs,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		RunMetadataLintCommand()
```

Comment:
A few places here and throughout, lets name everything as generic "lint" rather than metadata lint.